### PR TITLE
Add r-prettycode

### PR DESCRIPTION
**Pull request by xhochy**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/help-r This is ready for a review.

File: recipes/r-prettycode/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/prettycode_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/prettycode/prettycode_{{ version }}.tar.gz
+  sha256: 4f1274a4cc683a16823b0989aea2803053dec3cbd5caeb6bba64e67d312ee77b
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```
Comment (jdblischak) [inline recipes/r-prettycode/meta.yaml]: This package doesn't need to be compiled, so it should be a noarch recipe. Please add the line `noarch: generic` to this section.

If you didn't delete this line, you probably need to update `conda-build` to use the latest version of the skeleton (I confirmed that version 3.18.2 creates a noarch recipe.

Comment (xhochy) [reply recipes/r-prettycode/meta.yaml]: Weird, I have `conda-build=3.18.2` and it doesn't create a `noarch` recipe for me.

Comment (jdblischak) [reply recipes/r-prettycode/meta.yaml]: Ah, then you may need to update your local copy of the `conda_r_skeleton_helper`. It adds the flag `--use-noarch-generic`.

Comment (xhochy) [reply recipes/r-prettycode/meta.yaml]: I wasn't aware of `conda_r_skeleton_helper`, I was using plain `conda skeleton cran prettycode`. Pushed a new commit that uses this now.

Comment (jdblischak) [reply recipes/r-prettycode/meta.yaml]: Wow, well then take that as a complement! Your recipe looked so well-formatted that I assumed you had used it :-)

**Comment by jdblischak** [comment]
Thanks @xhochy!

**Pull request merged**